planning/behavior_path_planner/autoware_behavior_path_avoidance_by_lane_change_module/src/scene.cpp

```diff
@@ -273,16 +273,12 @@ double AvoidanceByLaneChange::calcMinAvoidanceLength(const ObjectData & nearest_
   return avoidance_helper_->getMinAvoidanceDistance(shift_length);
 }
 
-double AvoidanceByLaneChange::calcMinimumLaneChangeLength() const
+double AvoidanceByLaneChange::calc_minimum_dist_buffer() const
 {
-  const auto current_lanes = get_current_lanes();
-  if (current_lanes.empty()) {
-    RCLCPP_DEBUG(logger_, "no empty lanes");
-    return std::numeric_limits<double>::infinity();
-  }
-
-  return utils::lane_change::calculation::calc_minimum_lane_change_length(
-    getRouteHandler(), current_lanes.back(), *lane_change_parameters_, direction_);
+  const auto lc_length_and_dist_buffer =
+    utils::lane_change::calculation::calc_lc_length_and_dist_buffer(
+      common_data_ptr_, get_current_lanes());
+  return std::get<1>(lc_length_and_dist_buffer).min;
```

Comment:
(minor) I think you can do something like this which is a bit easier to read.
```suggestion
  const auto [_, dist_buffer] =
    utils::lane_change::calculation::calc_lc_length_and_dist_buffer(
      common_data_ptr_, get_current_lanes());
  return dist_buffer.min;
```